### ISSUE TITLE
Running ./bin/deploy-plugins.sh wipes out the publishing-filters

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#19037]
When dotCMS first starts up, it tries to copy the default publishing-filters from the `WEB-INF/classes` directory to the `./assets/server/publishing-filters` directory.  The issue arises if you run `bin/deploy-plugins.sh` before you start dotCMS for the first time.  The plugin deployer deletes out the WEB-INF/classes directory, which causes dotCMS to throw an error.

1. Download the release dotcms_5.3.5.tar.gz
2. unzip and set the db.properties/dotcms-cluster-config.properties appropriately
3. `cd` into the directory and run `./bin/deploy-plugins.sh` as you would if you have a plugin
4. start dotcms - `./bin/startup.sh run` and you will see an error as the `publishing-filters` fail to be copied.

This happens because the ant build.xml wipes out the WEB-INF/classes directory - line 1087 of bin\ant\build.xml
```
<delete dir="${tomcat.home}/webapps/ROOT/WEB-INF/classes/com"/>
```
Thanks Rahul Patel for finding this issue!

### Fix
Move the default publishing-filters to under the WEB-INF and not under the WEB-INF/classes folder.

### Workarounds:

1. Add the publishing filters to the assets dir after the initial dotCMS startup and restart dotCMS
2. Add the publshing filters to the ./ROOT directory of your static plugin:

![Screen Shot 2020-08-05 at 6 27 31 PM](https://user-images.githubusercontent.com/934364/89470741-7b099a00-d74a-11ea-90c1-b1e2b9ddced7.png)

**Comment by nollymar** [comment]
PR: https://github.com/dotCMS/core/pull/19056

**Comment by dotcms-corp** [comment]
The filters should be under a folder called publishing-
filters in the Web-inf directory, not under a class hierarchy

**Comment by rahulbpatel** [comment]
> ... 
> ### Workarounds:
> 1. Add the publishing filters to the assets dir after the initial dotCMS startup and restart dotCMS
> 2. Add the publshing filters to the ./ROOT directory of your static plugin:
> ...

@wezell regarding the workarounds how can we use the ROOT static config plugin if we can't call ./bin/deploy-plugins.sh first?

**Comment by wezell** [comment]
Because when you run **./bin/deploy-plugins.sh** the ones in your ROOT plugin will overwrite -or just place the files - where they need to be.

**Comment by rahulbpatel** [comment]
Okay, I see.  Thanks!

**Comment by bryanboza** [comment]
Fixed, tested on release-5.6.1 // Postgres // FF.
